packages/ui-components/src/components/switch-button/switch-button.scss

```diff
@@ -1,11 +1,39 @@
 @import '../../assets/styles/globals';
 
-$btn-height: 28px;
-$btn-width: 56px;
-$btn-radius: 4px;
-$sqr-size: 20px;
-$sqr-radius: 2px;
-$icon-size: 16px;
+$btn-height: 20px;
+$btn-width: 40px;
+$sqr-size: 16px;
+$sqr-padding-x: 2px;
+$icon-size: 12px;
+
+@function get-sqr-x-translation() {
+	@return $btn-width - $sqr-size - $sqr-padding-x * 2;
```

Comment:
If I set the `--btn-height` var the calculation don't return the correct value. 

Suggestion: 
```
:host {
	--btn-height: 20px;

	$btn-height: var(--btn-height);
}
```
after that use only the `$btn-height` for styling and calculations

Reply:
You're absolutely correct.

Fixing on the upcoming commit.